apps/site-nft/src/components/AppContainers/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styles from './styles.module.less';
+
+type AppContainerProps = {
+  className?: string,
+  shadow?: boolean,
+  noPadding?: boolean,
+}
+
+export const AppContainer : React.FC<AppContainerProps> = (props) =>
```

Comment:
I refactored AppContainer to just be a single type with arguments, instead of 3 different classes. It makes the composition a bit easier (no AppContainerWithShadowNoPadding)